pkg/platform/apicalls

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env bash
```

Comment:
Accidental commit?

Reply:
I found the usefulness of the script to be worth seeing if you'd be willing to keep it around. I can store it locally in my own dev environment if not.

Reply:
It would have to go into the `scripts` folder, but I think this is too situational. Don't think it makes sense to commit it.